stan/math/prim/mat/fun/cov_exp_quad.hpp

```diff
@@ -84,8 +86,8 @@ namespace stan {
                   Eigen::Dynamic, Eigen::Dynamic>
     cov_exp_quad(const std::vector<T_x1>& x1,
                  const std::vector<T_x2>& x2,
-                 T_sigma& sigma,
-                 T_l& l) {
+                 const T_sigma& sigma,
+                 const T_l& l) {
       using std::exp;
```

Comment:
You're right, I'm actually using `stan::math::exp`, so I've removed these `std::exp calls`